signature-specification.md

```diff
@@ -43,6 +43,7 @@ Signature Manifest Example
 - **`blobs`** (*array of objects*): This REQUIRED property contains collection of only one [artifact descriptor][artifact-descriptor] referencing signature envelope.
   - **`mediaType`** (*string*): This REQUIRED property contains media type of signature envelope blob. Following values are supported
     - `application/jose+json`
+    - `application/cose`
```

Comment:
similar to the suggestion I made above, I think this should be paired by the right `cose-type` parameter.

Reply:
[same as above](https://github.com/notaryproject/notaryproject/pull/148/files#r975829734), redundant/unnecessary